fdbserver/StorageMetrics.actor.h

```diff
@@ -380,6 +385,28 @@ struct StorageServerMetrics {
 		}
 	}
 
+	void getStorageMetrics(GetStorageMetricsRequest req, double bytesInputRate, int64_t versionLag) const {
```

Comment:
Why does it need this function?

Reply:
Yes, looks like I missed adding a delay here and that caused the actor to never yield. Adding a delay seems to have solved the infinite loop problem.

Reply:
Oh it's not needed for collecting metrics. I had Status json changes in the same branch. That needs this function. I had reverted Status.actor.cpp to narrow down the problem. Will attempt to re-instate that file now.